src/rulesets/rest/2022-05-25/json-api-rules/status-code-rules.ts

```diff
@@ -69,13 +69,13 @@ const post2xxCodes = new ResponseRule({
     response.statusCode.startsWith("2") &&
     rulesContext.operation.method === "post",
   rule: (responseAssertions) => {
-    const validPost2xxCodes = ["201"];
+    const validPost2xxCodes = ["201", "200"];
     responseAssertions.added(
       "support the correct 2xx status codes",
       (response) => {
         if (!validPost2xxCodes.includes(response.statusCode)) {
           throw new RuleError({
-            message: `expected POST response to only support 201, not ${response.statusCode}`,
+            message: `expected POST response to only support 201 or 200, not ${response.statusCode}`,
```

Comment:
`nit`: could we dynamically generate the string from `validPost2xxCodes`? Also, probably worth ordering them from low to high.